lib/osf-components/addon/components/tags-widget/styles.scss

```diff
@@ -1,7 +1,8 @@
 .TagsWidget.TagsWidget {
-    border: 0;
+    border: 0 !important;
     padding: 0;
     margin: 0;
+    margin-top: 10px;
```

Comment:
similar suggestion here, the 10px margin might not be appropriate for other invokations of this component. I think you can update `. tags-border` class in the submit/metadata to add the margin-top, especially now that we are adding splattributes to the TagsWidget

Reply:
Checked existing invocations of `<TagsWidget>` and from a cursory look, it looks like this isn't going to cause much issue either.